projects/testing/cypress/snapshot/plugin.ts

```diff
@@ -28,7 +28,7 @@ export async function tuiAddSnapshotPlugin(
         const {name, path, testFailure} = details;
         const possibleSnapshotPath = path
             .replace('screenshots', 'snapshots')
-            .replace(name, `${name}.snap`);
+            .replace(new RegExp(`${name}\\.(\\w+)$`, 'g'), `${name}.snap.$1`);
```

Comment:
oops, it does not work with `deep`-tests.
Their names have not-escaped `[`-character (

dont merge